_data/data.js

```diff
@@ -239,14 +241,25 @@ const downloadImage = async function (url, file) {
     });
 
     response.data.on('end', () => {
-      console.log(`Image received: ${path} (${hash.digest('hex')})`);
+      hashCode = hash.digest('hex');
+      console.log(`Image received: ${path} (${hashCode})`);
     });
     writer.on('finish', () => {
       console.timeEnd(label);
+      const hashFileName = `${hashCode}${extension}`;
+      const newPath = Path.join(IMAGE_DIRECTORY, hashFileName);
+      const newVirtualPath = Path.join(IMAGE_PATH, hashFileName);
+
+      FS.renameSync(`${path}`, `${newPath}`);
```

Comment:
You can make this lambda async and call `await FS.rename` here as well. Not a big deal though.